frontend/src/views/WheresMyGene/components/CellInfoSideBar/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { HTMLTable } from "@blueprintjs/core";
```

Comment:
Ah can we add a TODO to use SDS Table please? Let's stop using Blueprint for new features, so it's easier to remove Blueprint down the road 👌 Thank you!

Reply:
SDS table is too heavy weight for this table, we just needed a lightly styled html table and the blueprint table got us 95% of the way there

Reply:
Ah actually SDS Table is only available in SDS with MUIv5, so I just migrated it in my migration PR!

https://github.com/chanzuckerberg/single-cell-data-portal/pull/3734

<img width="735" alt="Screenshot 2022-12-13 at 6 09 21 PM" src="https://user-images.githubusercontent.com/6309723/207499197-e6a46a3f-5c09-45d9-b99c-5c803f2b7af3.png">